crates/workspace/src/workspace.rs

```diff
@@ -191,16 +192,16 @@ impl_actions!(
     ]
 );
 
-#[derive(Deserialize)]
+// #[derive(Deserialize)]
```

Comment:
This `Deserialize` impl doesn't appear to be used, AFAICT, and removing it allows us to avoid making `NotificationId` implement `Deserialize`.